lib/pbench/agent/utils.py

```diff
@@ -1,54 +1,29 @@
 import logging
 import os
 
-import colorlog
 
-
-def setup_logging(name=None, debug=False, logfile=None):
+def setup_logging(debug, logfile):
     """Setup logging for client
-    :param None: name of the python object
     :param debug: Turn on debug logging
     :param logfile: Logfile to write to
     """
-    if not name:
-        log = logging.getLogger()  # root logger
-    else:
-        log = logging.getLogger(name)
-
-    # Make sh logging a bit less verbose
-    logging.getLogger("sh").setLevel(logging.WARNING)
+    level = logging.DEBUG if debug else logging.INFO
+    fmt = "%(message)s"
 
-    # FIXME: since we dont do debugging level yet
-    log.setLevel(logging.INFO)
+    rootLogger = logging.getLogger()
+    rootLogger.setLevel(logging.NOTSET)
```

Comment:
Can you add a comment describing what `NOTSET` does?

Reply:
fixed